src/software/simulated_tests/simulated_test_fixture.cpp

```diff
@@ -86,14 +86,28 @@ void SimulatedTestFixture::setEnemyGoalie(RobotId goalie_id)
         ->setValue(static_cast<int>(goalie_id));
 }
 
-void SimulatedTestFixture::setPlay(const std::string &play_name)
+void SimulatedTestFixture::setAIPlay(const std::string &ai_play)
 {
     MutableDynamicParameters->getMutableAIControlConfig()
         ->mutableOverrideAIPlay()
         ->setValue(true);
     MutableDynamicParameters->getMutableAIControlConfig()
         ->mutableCurrentAIPlay()
-        ->setValue(play_name);
+        ->setValue(ai_play);
+}
+
+void SimulatedTestFixture::setRefboxPlay(const std::string &current_refbox_play,
```

Comment:
Shouldn't this be named something like `setRefboxGamestate` or similar? This isn't setting anything related to plays? Also can the interface be the raw enums? That would make the interface easier to use.

Reply:
I was mainly following the naming in dynamic params and visualizer. I can change it raw enums